migreringer/src/main/resources/db/postgres/defaultDS/1.0/V1.0_012__tabeller_for_fordel.sql

```diff
@@ -0,0 +1,174 @@
+    on mottatt_melding (journalpostid);
+
+create index uidx_mottatt_melding_soeknadid_tema_behandlingstema
+    on mottatt_melding (soeknad_id, tema, behandlingstema);
+
+create index idx_mottatt_melding_brevkode
+    on mottatt_melding (brevkode);
+
+
+create table journalpost_innsending
+(
+    journalpost_id       varchar(50)  not null
+        constraint pk_journalpost_innsending
+            primary key,
+    ytelse_type          varchar(50)  not null,
+    aktoer_id            varchar(50)  not null,
+    saksnummer           varchar(50)  not null,
+    status               varchar(20)  not null,
+    brevkode             varchar(50)  not null,
+    opprettet_tid        timestamp default LOCALTIMESTAMP,
+    endret_tid           timestamp,
+    innsendingstidspunkt timestamp,
+    payload              text
+);
+
+comment on table journalpost_innsending is 'Inneholder hvilke journalposter oversendes til saksystem.';
+comment on column journalpost_innsending.journalpost_id is 'Primary Key. Journalposten denne tabellen inneholder ekstra informasjon om.';
+comment on column journalpost_innsending.ytelse_type is 'Ytelsetype (fagsak ytelsetype) som journalpost gjelder.';
+comment on column journalpost_innsending.aktoer_id is 'Aktøren som journalpost gjelder for.';
+comment on column journalpost_innsending.saksnummer is 'Saksnummer som journalpost gjelder for.';
+comment on column journalpost_innsending.status is 'Status på søknaden: UBEHANDLET, INNSENDT';
+comment on column journalpost_innsending.brevkode is 'Brevkode (fra saf/joark) for journalpost.';
+comment on column journalpost_innsending.innsendingstidspunkt is 'Tidspunkt melding var originalt sendt inn til Nav (enten fra kildesystem) eller tidspunkt mottatt (dersom første ikke er angitt i melding)';
+comment on column journalpost_innsending.payload is 'payload fra journal';
+
+create unique index uidx_journalpost_innsending_jpid
+    on journalpost_innsending (journalpost_id);
+
+create index idx_journalpost_innsending_aktoerid
+    on journalpost_innsending (aktoer_id);
+
+create index idx_journalpost_innsending_saksnr
+    on journalpost_innsending (saksnummer);
+
+create index idx_journalpost_innsending_opprtid
+    on journalpost_innsending (opprettet_tid);
+
+create table produksjonsstyring_oppgave
+(
+    journalpost_id    varchar(50)                            not null
+        constraint pk_produksjonsstyring_oppgave
+            primary key,
+    aktoer_id         varchar(50),
+    ytelse_type       varchar(50),
+    fagsak_system     varchar(50),
+    behandlingstema   varchar(50),
+    oppgave_type      varchar(50)                            not null,
+    oppgave_id        varchar(50),
+    beskrivelse       text,
+    opprettet_tid     timestamp(3) default CURRENT_TIMESTAMP not null,
+    endret_tid        timestamp(3)
+);
+
+comment on table produksjonsstyring_oppgave is 'Inneholder ekstra IM-informasjon for angitt task.';
```

Comment:
betyr IM her innteksmelding? I så fall - trenger vi tabellen?

Reply:
Denne kommentaren er nok ikkje riktig. Eg ser at det er den samme som brukes for INNTEKTSMELDING_TASK_PROPS, så det er nok berre copy paste. Eg kan endre den til "Inneholder sporingsinformasjon for oppgaver opprettet i Gosys"